build.cake

```diff
@@ -130,6 +132,68 @@ void DoArchive(string runtime, DirectoryPath inputFolder, FilePath outputFile)
 }
 
 /// <summary>
+///  Generate the scripts which target the OmniSharp binaries.
+/// </summary>
+/// <param name="outputRoot">The root folder where the publised (or installed) binaries are located</param>
+void CreateRunScript(string outputRoot, string scriptFolder)
```

Comment:
Tasks cannot take arguments, so I might have to play around with explicit state variables to get it working with a separate task, but that can be confusing (difficult to infer the current state within any Task). 

In Cake you can have separate script files, but the way they are loaded is similar to the C include directive. This means that Tasks in separate files use the global state in the main file without having explicit references to it. I would prefer to keep Tasks in the main file, but we can move all helper functions into their own files (they only interact with the build via their arguments due to Mono restrictions anyway).

I could even port the helpers to ScriptCs, but that introduces an additional tooling to the system.

@david-driscoll I could also look into creating one based on the ScriptCs plugin. Do you think that is feasible for someone who is still not versed in the internals of OmniSharp?